recipes/MAPIT/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "MAPIT" %}
+{% set version = "1.4.8b0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sandialabs/MAPIT/archive/{{ version }}.tar.gz
+  sha256: 8c4ce88a592b2cbc3ff008435d256051562b5f213b91c7164173d16e958423cb
+
+build:
+  entry_points:
+    - MAPIT = MAPIT.GUI.MAPIT_main:launchMAPIT
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<3.12
```

Comment:
Please look at https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python and make the changes to satisfy the linter warnings.